lib/tasks/convert_event_bool_in_enum_genre.rake

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+# This namespace contain all tasks who adapt the data at changements of db structure
+namespace :convertion do # rubocop:disable Metrics/BlockLength
+  desc 'it set enum genre at event for mission who have the event boolean at true'
```

Comment:
it set`s` the :genre enum to the value of 2 (= event) for missions that have the :event boolean attribute set to true